### PR TITLE
feat: connectkit 2.0

### PR DESCRIPTION
**Pull request by chaichuanfa**
connectkit 2.0 docs

### CODE REVIEW

File: api-reference/connect/desktop/web.mdx [hunk 1]
```diff
@@ -1,241 +1,406 @@
...
-                    supportChains: [Ethereum, EthereumGoerli],
-                },
-            },
-        }}
-    >
-        <App />
-    </ModalProvider>
-  </React.StrictMode>
-)
+With Particle Connect installed, you'll need to open your `index` file and wrap your `App` component with `ConnectKitProvider`, imported from `@particle-network/connectkit`. Within `ConnectKitProvider`, you'll have a variety of configurations to set:
```
Comment (soos3d) [inline api-reference/connect/desktop/web.mdx:44]: Here, we mention using the `ConnectKitProvider` in the `index` of the app, but then we show something slightly different with the code snippet. Might be a bit confusing.

File: api-reference/connect/desktop/web.mdx [hunk 2]
```diff
@@ -1,241 +1,406 @@
...
+});
+
+// Wrap your application with this component.
+export const ParticleConnectkit = ({ children }: React.PropsWithChildren) => {
+    return <ConnectKitProvider config={config}>{children}</ConnectKitProvider>;
+};
 ```
 
+You can save this code snippet to a file named `connectkit.tsx`, and use `ParticleConnectkit` wrap your `App`.
+
```
Comment (soos3d) [inline api-reference/connect/desktop/web.mdx:118]: We say to use this as an independent component, but we discuss the index file in the previous section. 

In this format is not very clear that we suggest to keep `ConnectKitProvider` in its own component and then import it in your index file to wrap the app with it 👇

```tsx
// Wrap your application with this component.
export const ParticleConnectkit = ({ children }: React.PropsWithChildren) => {
    return <ConnectKitProvider config={config}>{children}</ConnectKitProvider>;
};
```

File: api-reference/connect/desktop/web.mdx [hunk 3]
```diff
@@ -1,241 +1,406 @@
...
+        width="600"
+        height="400"
+        src="https://www.youtube.com/embed/PIorEfDWRiE"
+        title="Supercharge Onboarding with Particle Network: Seamless Integration Guide"
+        alt="Supercharge Onboarding with Particle Network: Seamless Integration Guide"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+    ></iframe>
 </Frame>
```
Comment (soos3d) [inline api-reference/connect/desktop/web.mdx:406]: This video is probably outdated here?

File: api-reference/connect/desktop/web.mdx [hunk 4]
```diff
@@ -1,241 +1,406 @@
...
+            authTypes: ['email', 'phone', 'google', 'apple', 'twitter'],
+            fiatCoin: 'USD',
+            promptSettingConfig: {
+                promptMasterPasswordSettingWhenLogin: 1, // optional
+                promptPaymentPasswordSettingWhenSign: 1, // optional
+            },
+        }),
+    ],
+    // other options...
+});
```
Comment (soos3d) [inline api-reference/connect/desktop/web.mdx:157]: Also here we jump back to the configuration file without much context

File: api-reference/connect/desktop/web.mdx (hunk 3 above)
Comment (TABASCOatw) [reply api-reference/connect/desktop/web.mdx:406]: Yes

**Pull request closed without merging**